proposals/lip-0071.md

```diff
@@ -37,15 +37,15 @@ In the current [Lisk protocol][lip-0042], the rewards are the same for all activ
 
 ### Minting of the Rewards
 
-* After applying the block, a certain quantity of the native token of the blockchain is minted. The exact amount assigned to the block generator, i.e. the reward is calculated depending on the rules of the [Random module][lip-0046] and the [Lisk-BFT][lip-0058] protocol. Specifically, the reward is reduced for not participating in BFT block finalization or failure to properly reveal the random seed. This reduction happens exactly as before mentioned in the previous [Reward module][lip-0042]. After minting the block rewards, `updateSharedRewards `function is called from the [reward sharing mechanism][lip-rewardsharing] of the PoS module which locks the amount of the rewards which will be shared with the stakers.
+After applying the block, a certain quantity of the native token of the blockchain is minted. The exact amount assigned to the block generator, i.e. the reward is calculated depending on the rules of the [Random module][lip-0046] and the [Lisk-BFT][lip-0058] protocol. Specifically, the reward is reduced for not participating in BFT block finalization or failure to properly reveal the random seed. This reduction happens exactly as before mentioned in the previous [Reward module][lip-0042]. After minting the block rewards, `updateSharedRewards `function is called from the [reward sharing mechanism][lip-rewardsharing] of the PoS module which locks the amount of the rewards which will be shared with the stakers.
```

Comment:
[lip-rewardsharing] link is broken. I suggest updating this with [reward sharing mechanism of the Proof-of-Stake (PoS) module][link to function `updateSharedRewards`].

@vjaiman not sure when this was introduced, strange that we did not see this before..

Reply:
@gkoumout Yes, strange. May be during some update it got broken.

Reply:
Done.